infogami/core/helpers.py

```diff
@@ -108,7 +108,7 @@ def trimlist(x):
 
     def trimdict(x):
         y = {}
-        for k, v in x.iteritems():
+        for k, v in x.items():
```

Comment:
Used only once.  Python 2 will build a list in memory which will take up more RAM.  Python 3 will be fast.